scripts/reformat_cpp.sh

```diff
@@ -15,12 +15,12 @@ esac
 
 TOPDIR="$(readlink -f $SCRIPT_DIR/..)"
 
-if ! (clang-format --version 2>/dev/null | grep 10 >/dev/null); then
-    echo "clang-format 10 not available. Running it from a Docker image";
-    docker build -t proj_clang_format -f "$TOPDIR"/scripts/proj_clang_format/Dockerfile "$TOPDIR"/scripts/proj_clang_format
+if ! (clang-format --version 2>/dev/null | grep 15 >/dev/null); then
```

Comment:
If here you directly call `clang-format-15` (instead of the symbolic link `clang-format`) it is not mandatory to have the symbolic link to _that_ specific version. That allows the developer to have different versions of clang-format installed nicely. That should imply changes also in the script that finally calls clang-format.

Reply:
done in extra commit